CONTRIBUTING.rst

```diff
@@ -996,18 +996,22 @@ prepare such packages on your own easily.
 * Make sure you have ``setuptools`` and ``wheel`` installed in your python environment. The easiest way
   to do it is to run ``pip install setuptools wheel``
 
-* Enter the ``backport_packages`` directory
+* Run the following command:
+
+  .. code-blamazonock:: bash
```

Comment:
@kaxil - I am afraid you just revealed your super secret password :)

Reply:
haha. lol , have no idea what it is. 

oh wait, yeah I was trying `amazon` backport and copy pasted it between ``..code-block`` :D